XcodeServerSDKTests/RepositoryTests.swift

```diff
@@ -79,5 +79,35 @@ class RepositoryTests: XCTestCase {
         sshEnum = .LoggedInReadWrite
         XCTAssertEqual(sshEnum.toString(), "All logged in users can read and write")
     }
+    
+    // MARK: API Routes tests
+    func testGetRepositories() {
+        let expectation = self.expectationWithDescription("Get Repositories")
+        let server = self.getRecordingXcodeServer("get_repositories")
+        
+        server.getRepositories() { (repositories, error) in
+            XCTAssertNil(error, "Error should be nil")
+            XCTAssertNotNil(repositories, "Repositories shouldn't be nil")
+            
+            if let repos = repositories {
+                XCTAssertEqual(repos.count, 2, "There should be two repositories available")
+                
+                for (index, repo) in repos.enumerate() {
+                    XCTAssertEqual(repo.name, "Test\(index + 1)")
```

Comment:
And when it comes to the role of DVR, this is what I think:

Ideally, when we're writing out tests, we would always have a live Xcode Server accessible that would be setup a certain way and never change its state. So we'd write tests about what we know is true about this live Xcode Server and our tests would always call out and assert all the things we want to be true.

However, having a live server that never changes available at all times is pretty impractical for many reasons. This is where I see the value of DVR. It lets us snapshot a request + response with out setup Xcode Server and then it acts in place of our Xcode Server. Which we can take down and not have it sitting there in a constant state. But our tests should still work even if we ever delete our cassettes and put the original Xcode Server up.

You are correct, DVR also makes it very easy to create the stubs, which is mainly why I really wanted to use it. But I still see it as a tool that lets us take our live servers down and still run our tests unchanged.